build.fsx

```diff
@@ -243,7 +244,7 @@ Target.create "RunTests" (fun _ ->
         use waiter = new ManualResetEvent(false)
         let serverProjectDir = Path.GetDirectoryName(serverProject)
         let projectName = Path.GetFileNameWithoutExtension(serverProject)
-        let serverExe = "bin" </> "Release" </> "netcoreapp2.1" </> (projectName + ".dll")
+        let serverExe = "bin" </> "Release" </> "netcoreapp3.1" </> (projectName + ".dll")
```

Comment:
I decided to upgrade .NET Core version in samples and tests because it was needed for less complication in dependencies with System.Net.HttpExtensions.